symfony/plugins/orangehrmRecruitmentPlugin/lib/form/AddJobVacancyForm.php

```diff
@@ -23,6 +23,17 @@ class AddJobVacancyForm extends BaseForm {
     private $vacancyService;
     private $vacancyId;
     private $jobTitleService;
+    private $employeeService;
+
+    /**
+     * @return EmployeeService
+     */
+    public function getEmployeeService() {
```

Comment:
Improve the doc bloc. 
Give a brief description of what the functions does. example : This method will return an instance of Employee Service.
Add a Setter Method